docs/contributing.md

```diff
@@ -1,2 +1,73 @@
-```{include} ../CONTRIBUTING.md
-```
\ No newline at end of file
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+## Types of Contributions
+
+### Report Bugs
+
+If you are reporting a bug, please include:
+
+- Your operating system name and version.
+- Any details about your local setup that might be helpful in troubleshooting.
+- Detailed steps to reproduce the bug.
+
+### Fix Bugs
+
+Look through the GitHub issues for bugs. Anything tagged with "bug" and "help
+wanted" is open to whoever wants to implement it.
+
+### Implement Features
+
+Look through the GitHub issues for features. Anything tagged with "enhancement"
+and "help wanted" is open to whoever wants to implement it.
+
+### Write Documentation
+
+You can never have enough documentation! Please feel free to contribute to any
+part of the documentation, such as the official docs, docstrings, or even
+on the web in blog posts, articles, and such.
+
+### Submit Feedback
+
+If you are proposing a feature:
+
+- Explain in detail how it would work.
+- Keep the scope as narrow as possible, to make it easier to implement.
+- Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+## Get Started!
+
+Ready to contribute? Here's how to set up `aiod_sdk` for local development.
```

Comment:
After I renamed the package to `aiondemand` I left behind some places with different names, such as `aiod_sdk` here. I think we should replace it by the correct name in these files in this PR, unless you want to open another PR to review all the content of these auto generated files.

Reply:
I'll update them in a follow up PR. It's easier to see the change. I would personally also prefer to change the recommendation from `poetry` to `pip`. I don't really see a reason to recommend `poetry`. Especially for a library I think defaulting to `^` version matching is not great. And it generally doesn't really do much that `pip` and `pip-tools` already cover. Please educate me if I'm wrong :)

Reply:
This is fine. Honestly, I followed the guidelines on [py-pkg](https://py-pkgs.org/03-how-to-package-a-python) and they suggested Poetry, which I found handy for setting up the project structure and publishing it to PyPi. As I used it, the recommendations were automatically generated. So, we can move forward with pip as well. :)